ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -294,6 +296,36 @@ sub init_globals {
     return \%globals;
 }
 
+sub process_options {
+    my ($globals, $opts) = @_;
+
+    @{$globals->{Podpath}}  = split(":", $opts->{podpath}) if defined $opts->{podpath};
```

Comment:
```suggestion
    $globals->{Podpath}     = [ split(":", $opts->{podpath}) if defined $opts->{podpath} ];
```

I don't _think_ it's intentional that right now you're keeping the same underlying array if this is ever called twice, which is just a tiny bit closer to allowing action at a distance.  Also, un-special-casing this would allow you to slot it down in alpha order.  This is a minor thing, though.

Reply:
> I don't _think_ it's intentional that right now you're keeping the same underlying array if this is ever called twice, which is just a tiny bit closer to allowing action at a distance. Also, un-special-casing this would allow you to slot it down in alpha order. This is a minor thing, though.

I took this approach because in the current production (v5.34.0) release, `ext/Pod-Html/lib/Pod/Html.pm` has this:
```
553 
554     @Podpath  = split(":", $opt_podpath) if defined $opt_podpath;
555 
```
When refactoring a file to replace 20+ lexical variables defined outside of the principal subroutine with `%globals` (which in the next p.r. will become a Pod::Html object), I was taking care to just change one thing at a time.  Hence, the syntax above rather than your suggestion.  By the same rationale, I have not yet changed any of the `Initialcap` spelling of those lexicals (which are now keys within `%globals`) -- even though if I were starting from scratch they would be all `lowercase`.  Since changing those keys from `Initialcap` to `lowercase` would expand the size of the diff without improving functionality, I'd prefer to do that after we've got our Pod::Html object in place.

In any case, when I tried out your suggested replacement code, I got:
```
!perl -Ilib -c lib/Pod/Html.pm
syntax error at lib/Pod/Html.pm line 328, near ") if"
Global symbol "$globals" requires explicit package name (did you forget to declare "my $globals"?) at lib/Pod/Html.pm line 330.
Global symbol "$opts" requires explicit package name (did you forget to declare "my $opts"?) at lib/Pod/Html.pm line 330.
...
```
Perhaps this is what you were aiming for?
```
-    @{$globals->{Podpath}}  = split(":", $opts->{podpath}) if defined $opts->{podpath};
+    $globals->{Podpath}   = (defined $opts->{podpath})
+                            ? [ split(":", $opts->{podpath}) ]
+                            : [];
```

Reply:
I just meant to put the `]` before the `if` clause, and not after.  I think your version, or the code as it is, are fine.